### PR TITLE
PR for Card Component and Getting Started Page

### PR DESCRIPTION
**Pull request by manojasarsa**
Added Card Component and Getting Started Page

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+@import url("/docs/main.css");
+
+.card {
+    min-height: fit-content;
+    width: 60%;
+    justify-content: flex-start;
+    box-shadow: 2px 2px 10px grey;
```
Comment (mimansha-swarup) [inline components/card/card.css]: you can make a variable of shadow s

**Review by mimansha-swarup — APPROVED** [review]
looks good to me 👍

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    margin: 1rem 1rem 0 1rem;
+}
+
+.card_image {
+    display: flex;
+}
+
+.card_vertical {
+    display: flex;
+    flex-direction: column;
```
Comment (vaibhav18matere) [inline components/card/card.css]: can add seperate css class `flex-col` as `display:flex` `flex-direction:column` is repeating

**Review by vaibhav18matere — APPROVED** [review]

File: components/getting-started.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    </div>
+
+    <!-- Main Content -->
+
+    <div class="doc_content">
+
+        <div class="doc_main_container">
+            <h2 class="doc_main_head">Getting Started</h2>
+            <p class="doc_main_desc">
+                <span class="span">ZERO+1 UI</span> is an CSS component library, which helps Developers to use best styles and components, by using pre-defined classnames. It will quickly build your Dream Project because we take care of your Designs.
```
Comment (vaibhav18matere) [inline components/getting-started.html]: is **a** CSS library

**Review by vaibhav18matere — APPROVED** [review]

File: components/card/card.css (hunk 1 above)
Comment (manojasarsa) [reply components/card/card.css]: Oh I forget that. Thanks. for reminding!

File: components/card/card.css (hunk 2 above)
Comment (manojasarsa) [reply components/card/card.css]: Done, Vaibhav!

**Pull request merged**